src/extensions/score_metamodel/checks/id_contains_feature.py

```diff
@@ -37,6 +38,7 @@ def id_contains_feature(app: Sphinx, need: NeedsInfoType, log: CheckLogger):
 
     # Get the part of the string after the first two underscores: the path
     feature = parts[1]
+    featureparts = re.split(r"_-", feature)
```

Comment:
```suggestion
    featureparts = re.split(r"[_-]", feature)
```
I think it needs to be this, otherwise it will look for '_-' instead of '-' or '_'